### ISSUE TITLE
o2w crashes when image has alt text

### ISSUE THREAD

**Issue by regalialong** [lun-4/obsidian2web#37]
Obsidian syntax for alt text is `![[file.jpg|alt text]]`, img element shows that they don't just discard the text but actually embed it as alt text.

![Picture of img element with the alt text "it's a puppy!!!"](https://github.com/lun-4/obsidian2web/assets/126194895/a4ec7b24-134a-4f1f-92b6-5ad05757bf2d)

This isn't as critical within a Vault as that's for personal use, but on the web, it'd be preferable to have the alt text embedded both for accessibility and for users who either refuse to load or cannot load images for some reason.

obsidian2web doesn't understand this syntax and tries to parse it literally as link.
```
debug(obsidian2web_util): captured regex at offset 0
error(obsidian2web_processors): referenced name: puppy.jpg|it's a puppy!!! not found
error: InvalidLinksFound
???:?:?: 0x3b8bc8 in ??? ()
???:?:?: 0x3b8bd6 in ??? ()
???:?:?: 0x3b8c39 in ??? ()
???:?:?: 0x3b94a5 in ??? ()
???:?:?: 0x380281 in ??? ()
```
Reproducing this should be straightforward:
Create document with alt text image, o2w explodes
